[Service Bus] Add browser support (#5128)
Azure/azure-sdk-for-js: sdk/servicebus/service-bus/changelog.md

> # 2019-09-23 1.1.0

Reviewer: ```suggestion
# 2019-09-24 1.1.0
```

> - Add browser support. Authentication with `@azure/identity` is not supported yet - use a connection string instead.

Reviewer: Users of this library are not aware of what `@azure/identity` is. If they have been using AAD, then that would be by using the `@azure/ms-rest-nodeauth` library.

```suggestion
- Add browser support. Authentication using Azure Active Directory credentials is not supported yet - use a connection string instead.
```

# 2019-09-23 1.1.0

> - Update library to support running all Service Bus operations in browser.

Reviewer 2: Add browser support. Authentication with `@azure/identity` is not yet supported &ndash; use a connection string instead.

Author: Updated

> # 2019-09-19 1.1.0

Reviewer: ```suggestion
# 2019-09-22 1.1.0
```

We mostly will not be releasing on a Friday

- Update library to support running all Service Bus operations in browser.

> [PR 5128](https://github.com/Azure/azure-sdk-for-js/pull/5128) related to [issue 3373](https://github.com/Azure/azure-sdk-for-js/issues/3373)

Reviewer: Add a note about not supporting AAD in browser yet and that using a connection string is the only way to use the library
